### ISSUE TITLE
Allow passing flags to grep

### ISSUE THREAD

**Issue by avahe-kellenberger** [galatolofederico/pso#6]
In some cases I'd like to be able to pass extra arguments to grep, such as `-i` for case insensitive searching. It would be nice to have a way to pass arbitrary flags for matches!

**Comment by galatolofederico** [comment]
Hi, thanks for the issue! How are you suggesting to implement this feature? can you provide some example of how it should work?

**Comment by avahe-kellenberger** [comment]
Haven't entirely thought it through, but maybe just allowing an additional "field" for grep flags. E.g:

`mpv %s:-i:.*\.mp4`

Where -i is the case insensitivity flag

**Comment by galatolofederico** [comment]
Yes i think that would work, we can make it more general using `grep -E` as default command and leave the user the ability to set a custom command to parse the rule like `grep -i`, `awk` , etc. What do you think?

**Comment by avahe-kellenberger** [comment]
That sounds good to me 👍
